app/views/comments/index.html.erb

```diff
@@ -1,26 +1 @@
-<h1>Comments</h1>
-
-<table class="table">
-  <% @comments.each do |comment| %>
-  <tr id="c<%= comment.cid %>">
-    <td style="width:80%;">
-      <i class="fa fa-comment-alt"></i> 
-      <% if comment.author # some legacy comments have no author %>
-        <a href="/profile/<%= comment.author.name %>"><%= comment.author.name %></a> 
-      <% end %>
-      <span style="color:#aaa;"><%= t('home._comments.commented') %> 
-        <% if comment.parent %>
-          on <a href="<%= comment.parent.path %>"><%= comment.parent.title %></a>
-        <% end %>:
-      </span>
-      "<%= raw strip_tags(sanitize(RDiscount.new(comment.comment).to_html)).truncate(150) %>" | 
-      <% if comment.parent %>
-        <a href="<%= comment.parent.path %>#c<%= comment.cid %>"><%= t('home._comments.read_more') %> &raquo;</a>
-      <% end %>
-    </td>
-    <td><%= distance_of_time_in_words(comment.created_at, Time.current, false, :scope => :'datetime.time_ago_in_words') %></td>
-  </tr>
-  <% end %>
-</table>
-
-<%= will_paginate @comments, :renderer => BootstrapPagination::Rails %>
+<%= render partial: 'comments/comments' %>
```

Comment:
@jywarren Moved the comments view to a separate partial as the comments action in the user controller was previously using the `home/comments` partial which was broken after the previous changes you made. Since rendering the `comments/index` template also included the header and footer so the only option was to make a separate template for it.